myriad-scheduler/src/main/resources/myriad-config-default.yml

```diff
@@ -1,14 +1,14 @@
-mesosMaster: 10.0.2.15:5050
+mesosMaster: 172.31.45.229:5050
 checkpoint: false
 frameworkFailoverTimeout: 43200000
-frameworkName: MyriadAlpha
+frameworkName: Myriad
```

Comment:
@bgulla Looks like these config changes were made to match your specific environment. Is that correct? If yes, can you please remove this file from the PR.